src/formElementPlugins/fileUpload/src/components/FileDisplayWIthDescription.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react'
+import { Icon, Grid, List, Image, Message, Loader, Input } from 'semantic-ui-react'
+import { useLanguageProvider } from '../../../../contexts/Localisation'
+import { FileDisplayProps } from './FileDisplay'
+import prefs from '../../config.json'
+
+interface FileDisplayDescriptionProps extends FileDisplayProps {
+  description?: string
+  updateDescription: (uniqueId: string, value: string, key: string) => void
+}
+
+export const FileDisplayWithDescription = ({
+  file,
+  onDelete,
+  downloadUrl,
+  updateDescription,
+}: FileDisplayDescriptionProps) => {
+  const { getPluginStrings } = useLanguageProvider()
+  const strings = getPluginStrings('fileUpload')
+  const { loading, error, errorMessage, filename, fileData, key } = file
+  const [description, setDescription] = useState<string>(fileData?.description ?? '')
+  return (
+    <List.Item>
+      {fileData && (
```

Comment:
should remove this check here `   {fileData && (` otherwise errors are not shown

Reply:
Cool -- b66bf0e4